tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -10,25 +10,29 @@ def test_fields(self):
         uut = DocumentationComment("my doc",
                                    "c",
                                    "default",
+                                   5,
                                    ("/**", "*", "*/"),
                                    (25, 45))
 
         self.assertEqual(uut.documentation, "my doc")
         self.assertEqual(uut.language, "c")
         self.assertEqual(uut.docstyle, "default")
+        self.assertEqual(uut.indent, 5)
         self.assertEqual(str(uut), "my doc")
         self.assertEqual(uut.marker, ("/**", "*", "*/"))
         self.assertEqual(uut.range, (25, 45))
 
         uut = DocumentationComment("qwertzuiop",
                                    "python",
                                    "doxygen",
+                                   0,
```

Comment:
if you use `0` as default, then you can leave this here and test for the default value :+1: